CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ v1.6.1 - unreleased
 - Enhancement: Stop using static assets, but build them using Webpack
 - Enhancement: Create an Authentication Token with IPv6 address restriction ([#829](https://github.com/Mailu/Mailu/issues/829))
 - Enhancement: Missing wildcard option in alias flask command ([#869](https://github.com/Mailu/Mailu/issues/869))
+- Enhancement: Fixed hardcoded antispam and antivirus host addresses ([#978](https://github.com/Mailu/Mailu/issues/978))
```

Comment:
This should be in towncrier: https://mailu.io/master/contributors/guide.html#changelog